src/cmd.rs

```diff
@@ -1919,6 +1942,97 @@ command!(
     }
 );
 
+fn print_statefulsets(
+    list: StatefulSetList,
+    regex: Option<Regex>,
+    writer: &mut ClickWriter,
+) -> StatefulSetList {
+    let mut table = Table::new();
+    table.set_titles(row!["####", "Name", "Desired", "Current", "Ready"]);
+    let statefulsets_specs = list.items.into_iter().map(|statefulset| {
+        let mut specs = Vec::new();
+        specs.push(CellSpec::new_index());
+        specs.push(CellSpec::new_owned(
+            val_str("/metadata/name", &statefulset, "<none>").into_owned(),
+        ));
+        specs.push(CellSpec::new_owned(format!(
+            "{}",
+            val_u64("/spec/replicas", &statefulset, 0)
+        )));
+        specs.push(CellSpec::new_owned(format!(
+            "{}",
+            val_u64("/status/currentReplicas", &statefulset, 0)
+        )));
+        specs.push(CellSpec::new_owned(format!(
+            "{}",
+            val_u64("/status/readyReplicas", &statefulset, 0)
+        )));
+        (statefulset, specs)
+    });
+
+    let filtered = match regex {
+        Some(r) => ::table::filter(statefulsets_specs, r),
+        None => statefulsets_specs.collect(),
+    };
+
+    ::table::print_table(&mut table, &filtered, writer);
+
+    let final_statefulsets = filtered
+        .into_iter()
+        .map(|statefulset_spec| statefulset_spec.0)
+        .collect();
+
+    StatefulSetList { items: final_statefulsets }
+}
+
+command!(
+    StatefulSets,
+    "statefulsets",
+    "Get statefulsets (in current namespace if set)",
+    |clap: App<'static, 'static>| clap.arg(
+        Arg::with_name("show_label")
+            .short("L")
+            .long("labels")
+            .help("Show statefulsets labels")
+            .takes_value(true)
+    ).arg(
+        Arg::with_name("regex")
+            .short("r")
+            .long("regex")
+            .help("Filter statefulsets by the specified regex")
+            .takes_value(true)
+    ),
+    vec!["rs", "statefulsets"],
```

Comment:
this should have `ss` as an alias I think, not `rs`

Reply:
oh, I missed this one. Good catch.